README.md

```diff
@@ -79,24 +99,38 @@ var express = require("express");
 var app = express();
 
 // Your code starts here...
-
+app.get("/", function(req, res){
+  console.log("Shows the app-welcome page")
+  res.send("app-welcome");
+});
+
+app.post("/", function(req, res){
+  res.send("Create");
+});
+
+app.put("/:id", function(req, res){
+  res.send("Update");
+});
+
+app.delete("/:id", function(req, res){
+  res.send("Destroy");
+});
 ```
 
-```js
-// Your answer...
-```
 ### Question #7
 
 Describe the differences between Express and Rails as backend frameworks.
 
-```text
+```
+Rails is a framework build on ruby that follows strict convention in terms of the syntax and organizational structure that builds the app. Rails starts with a good amount of code that people can build off of. Express on the other hand practically starts from scratch and it is up to the developer to build the routing system and links. Also, CRUD functionality is implemented differently in both. That is, Rails uses migrations whereas Express installs node modules.
 
 ```
 
 ### Question #8
 
-What is the importance of using body-parser in our express application for post requests? 
+What is the importance of using body-parser in our express application for post requests?
 
-```js
+```
+Body parser requests bodies of code that support json objects. It extracts the entire body, which allows the app to see the entire contents of the body before parsing it. In terms of POST params, it allows express to grab information from the POST, create routes, and send information.
```

Comment:
The body-parser gathers the form information from the post and provides it to Express request as `body`.